dashboard/src/main/resources/webapp/static/lib/angular-topology-editor/canvas.js

```diff
@@ -101,7 +101,8 @@ Graph.Node.popovercontent = function(i) {
 };
 
 var Canvas = (function() {
-
+    var LOG = Log.getLogger("Canvas").setLevel(Log.INFO);
```

Comment:
I know, but logging is IMO a minor feature heavily used across libraries, anyway if you want I could try to change it back.